sparkles/core.py

```diff
@@ -884,6 +884,23 @@ def check_guide_count(self):
                 'caution',
                 f'{obs_type} with more than {bright_cnt_lim} stars brighter than 6.1.')
 
+        n_guide = len(self.guides)
+
+        # Different number of guide stars than requested
+        if n_guide != self.n_guide:
+            self.add_message(
+                'caution',
+                f'{obs_type} with {n_guide} guides but {self.n_guide} were requested')
+
+        # Caution for any "unusual" guide star request
+        typical_n_guide = 5 if self.is_OR else 8
+        if self.n_guide != typical_n_guide:
+            msg = f'{obs_type} with {n_guide} guides requested but {typical_n_guide} is typical'
+            if self.is_OR and n_guide == 4:
+                msg += f' (likely MON star, check OR list)'
```

Comment:
Not feedback on the PR, but I note that we don't always have an OR list to check.

Reply:
Sure, but in practice if you actually see this message on your screen there almost surely be a corresponding OR list somewhere.

Reply:
Sure, there may be a list, but I *think* that there are no entries in the list for say, pool targets, until after prelim review?  Or maybe I just haven't tried hard enough to find it.   Just an aside.